deployment/sandbox-v2/docs/idacert.md

```diff
@@ -20,212 +21,126 @@
 	```
 	{
+     "id": "string",
+     "version": "string",
+     "requesttime": "2023-05-30T04:36:36.006Z"
+     "metadata": {},
+     "request": {
+     "applicationId": "IDA",
+     "referenceId": "PUBLIC_KEY",
+     "certificateData": "{copied certificate from step-II}"
+  }
+  }
+	```
+ 
+ P.S : The process of uploading IDA Zero-Knowledge certificates should only take place once per environment.
+ 
+  B)for IDA mpartner-default-auth partner certificate exchange, below are the steps
+      
+   * 1. Authenticate yourself and get authorization token from authmanager swagger
+      SWAGGER URL:- ```https://minibox.mosip.net/v1/authmanager/swagger-ui.html#/authmanager/clientIdSecretKeyUsingPOST ```
+       ```
+	{
 	  "id": "string",
 	  "metadata": {},
 	  "request": {
-	    "certificateData": "-----BEGIN CERTIFICATE-----\nMIIDTjCCAjagAwIBAgIEYFrxXTANBgkqhkiG9w0BAQsFADBpMQswCQYDVQQGEwJJ\nTjESMBAGA1UECAwJS2FybmF0YWthMRIwEAYDVQQHDAlCYW5nYWxvcmUxHjAcBgNV\nBAoMFW1wYXJ0bmVyLWRlZmF1bHQtYWJpczESMBAGA1UEAwwJYWJpcy1yb290MB4X\nDTIxMDMyNDA3NTkyNVoXDTIyMDMyNDA3NTkyNVowaTELMAkGA1UEBhMCSU4xEjAQ\nBgNVBAgMCUthcm5hdGFrYTESMBAGA1UEBwwJQmFuZ2Fsb3JlMR4wHAYDVQQKDBVt\ncGFydG5lci1kZWZhdWx0LWFiaXMxEjAQBgNVBAMMCWFiaXMtcm9vdDCCASIwDQYJ\nKoZIhvcNAQEBBQADggEPADCCAQoCggEBANkwlDzNZTBi1fBF4GU4qFAJ3S+Ca0Kf\ngfvg93rQlZ5LBTnZFwAxpCZtGHYb7vkqM9e7adYGC48EPWI0A+48QmF3Z6vSBXg9\nKckINa/vFCTEYrctMHS8CcBjWBf9agJq4+wWqNu8sYHD9pOzDf1uMbQJTI5VvgGx\nv890pZrXdIrR4MPTLB0rkl2sVOqbG7bts0Eqh8TO86126CDzoDrtBCj3RBP/j/dg\nBmz7LWFkG6/by+mXzdZcS46v7P/Q366WrDbMCCtjKIRAA0HQD3vdKT0V03Eiw/EU\nVxVh9sdbkO5h/T8VWI7ghEjr4PpJXPYWRbVlt6uPDpbX+yEiOWG/SsMCAwEAATAN\nBgkqhkiG9w0BAQsFAAOCAQEAEj42FlN8LnNPv3iWttydxm9kEJemyJdw8nPLCC4y\nxigXrcxPgNcoJiDBXLIAwhTmPK1hdn/BndAeUsX8mauuzf4V7Ydw1a999s8Vsj8S\nOLa8voXAE2sjdYZm0cYID0y/ak3+ZrKqCXP6bcmPOLz2plnGJB7TUQ+d8gZXsLA6\nCoopaJOlNM4jPNbX/k30vfFmyrXm2++5stErrSOix25J79DGdmJH896/pmGmB60/\nXGnpyESrVTbhTE+cx0gDHdq5T47qHcXM6CVuH/uYNy5iLCaBRzVQ043gFj3ioym1\nnZ60dsvdG8nEENBu9SzN3Mn24pz0BQ99Qn5ymsQwYAEeDQ==\n-----END CERTIFICATE-----\n",
-	    "partnerDomain": "AUTH"
+	    "appId": "ida",
+	    "clientId": "mosip-ida-client",
+	    "secretKey": "abc123"
 	  },
-	  "requesttime": "2021-03-24T08:24:13.349Z",
+	  "requesttime": "2018-12-10T06:12:52.994Z",
 	  "version": "string"
 	}
 	```
-  * 4. Get the IDA certificate data from the below URL, Copy it and use it for certificate exchange in the next step.
-        ```https://minibox.mosip.net/idauthentication/v1/internal/getCertificate?applicationId=IDA```
 
-  * 5. Upload IDA certificate from the above request in the below SWAGGER URL:- ```https://minibox.mosip.net/v1/partnermanager/swagger-ui.html#/Partner%20Service%20Controller/uploadCACertificat$       
-  	 ```
-        {
+
+  * 2. Get the ROOT certificate data from the below URL.
+       
+        ```https://minibox.mosip.net/idauthentication/v1/internal/getCertificate?applicationId=ROOT```
+       make sure to copy the certificate value in the response from the above request.
```

Comment:
Ensure to copy the certificate value in the response from the above request.

(leave a space in between sentences)